keel-core/src/main/kotlin/com/netflix/spinnaker/keel/pause/ActuationPauser.kt

```diff
@@ -66,59 +69,55 @@ class ActuationPauser(
   fun pauseApplication(application: String) {
     log.info("Pausing application $application")
     pausedRepository.pauseApplication(application)
-    publisher.publishEvent(ApplicationActuationPaused(application))
+    publisher.publishEvent(ApplicationActuationPaused(application, "Application $application paused", clock))
```

Comment:
Seems like that message could be generated inside the constructor rather than having to be passed in since it looks like it's always going to be the same

Reply:
Good catch, will fix.